src/Maestro/Maestro.Web/Api/v2018_07_16/Controllers/AssetsController.cs

```diff
@@ -123,8 +146,14 @@ public async Task<IActionResult> AddAssetLocationToAsset(int assetId, [Required]
                 new AssetLocation(assetLocation));
         }
 
+        /// <summary>
+        ///   Removes an existing <see cref="AssetLocation"/> from an <see cref="Asset"/>.
+        /// </summary>
+        /// <param name="assetId">The id of the <see cref="Asset"/> to remove the <see cref="AssetLocation"/> from.</param>
+        /// <param name="assetLocationId">The id of the <see cref="AssetLocation"/> to remove.</param>
+        /// <returns></returns>
```

Comment:
Let's remove the empty <returns></returns>. :-)

Reply:
this all got autogenerated when I typed `///`. I will remove the empty `<returns>` because those are useless. I would like to leave the parameters and the warning that complains when they don't match so the we don't forget to document things when we add/change parameters.